spec/app_support/order_row_importer_spec.rb

```diff
@@ -22,16 +22,19 @@
   end
   let(:user) { create(:user) }
 
-  let(:row) {{
-    "Netid / Email" => username,
-    "Chart String" => chart_string,
-    "Product Name" => product_name,
-    "Quantity" => quantity,
-    "Order Date" => order_date,
-    "Fulfillment Date" => fulfillment_date,
-    "Errors" => errors,
-    "Note" => notes,
-  }}
+  let(:row) do
+    ref = {
+      "Netid / Email" => username,
+      "Chart String" => chart_string,
+      "Product Name" => product_name,
+      "Quantity" => quantity,
+      "Order Date" => order_date,
+      "Fulfillment Date" => fulfillment_date,
+      "Errors" => errors,
+      "Note" => notes,
+    }
+    CSV::Row.new(ref.keys, ref.values)
```

Comment:
Faking a row with a hash no longer worked because i needed to use CSV::Row#headers to validate headers.